src/content.js

```diff
@@ -59,38 +59,40 @@ function isWindowOpen(a) {
 }
 
 function getDownloadURL(anchorTag) {
-  const s = anchorTag.getAttribute('onclick')
-  const first = s.indexOf("'") + 1
-  const second = s.indexOf("'", first)
-  return s.substring(first, second)
+  const s = anchorTag.getAttribute('onclick');
+  const first = s.indexOf("'") + 1;
+  const second = s.indexOf("'", first);
+  return s.substring(first, second);
 }
 
 function getSchool() {
-  const schoolURL = window.location.href
-  const end = schoolURL.indexOf('.ca')
-  const start = schoolURL.lastIndexOf('.', end - 1) + 1
-  return schoolURL.substring(start, end)
+  const schoolURL = document
+    .getElementsByTagName('link')[0]
+    .getAttribute('href');
```

Comment:
The fix was reverted in the process of fixing the semicolons. Please add it back.

Reply:
Sorry about that
I just added it back